django_netjsonconfig/widgets.py

```diff
@@ -27,8 +27,8 @@ def media(self):
     def render(self, name, value, attrs={}):
         attrs['class'] = 'vLargeTextField jsoneditor-raw'
         html = """
-<a href="#advanced_editor" class="button json-editor-btn-edit normal-mode">{0}</a>
```

Comment:
Maybe you could bind it to the button "jsoneditor-poweredBy" (the name doesn't reflect anymore its function).